msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1132,12 +1133,13 @@ private void setupTelemetry(@NonNull final Context context,
     }
 
     @Override
-    public String getOrgIdFamilyRefreshToken(@NonNull final String identifier) throws MsalClientException {
+    public TokenShareResult getOrgIdFamilyRefreshTokenWithMetadata(@NonNull final String identifier) throws MsalClientException {
```

Comment:
Is this public API change?   I'd talk to Teams to see if they are will to take this as a change without incrementing the major version number... unless we can bundle this with the prompt and other breaking changes.

Reply:
Resolved offline